api/desecapi/serializers.py

```diff
@@ -579,18 +583,11 @@ def get_fields(self):
         return fields
 
     def validate_name(self, value):
-        self.raise_if_domain_unavailable(value, self.context['request'].user)
+        user = self.context['request'].user if not isinstance(self.context['request'].user, AnonymousUser) else None
```

Comment:
Why not pass `AnonymousUser` to `is_registerable`? I feel like that function should be able to handle that.

Reply:
This would imply that the whole `Domain` model can deal with `AnonymousUser` values for `owner`, but it doesn't. The problem is that when you filter a queryset for an owner value, the ORM produces an error when the filter value is `AnonymousUser`. We could now fix that specifically in `is_registrable()`, but I think that's very inconsistent with other functionality in the model then.

We can also fix `Domain.__init__` to set `owner = None` in this case. That would solve the problem. What do you think?

Reply:
I'm fine with it either way. I expect that we won't have a lot of `AnonymousUser` domain owner cases, so I don't want to over-engineer it

Reply:
fixed